tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/management/ActiveMQServerControlUsingCoreTest.java

```diff
@@ -37,35 +36,45 @@ public ActiveMQServerControlUsingCoreTest(boolean legacyCreateQueue) {
    }
 
    @Override
-   @Test
+   @Ignore
    public void testListProducersAgainstServer() throws Exception {
-      // have to disable this test, as it's dealing with producers objects.
-      // the test itself will be using a producer to manage the server.
-      // so the test will include noise and it might fail occasionally
-      Assume.assumeTrue(false);
    }
-   // ActiveMQServerControlTest overrides --------------------------
+
+
+   @Ignore
+   @Override
+   public void testListSessions() throws Exception {
+      // similarly to testListProducersAgainstServer test,
```

Comment:
Maybe explain it in the first test testListProducersAgainstServer as it was originally, and then reference that from this one with just this first sentence...rather than reference the earlier test when it would now have no explanation why it is ignored after these changes.